cms/migrations/0029_setup_course_program_index_pages.py

```diff
@@ -104,7 +104,10 @@ def unnest_detail_and_delete_index_pages(apps, schema_editor):
 
 class Migration(migrations.Migration):
 
-    dependencies = [("cms", "0028_course_program_index_pages")]
+    dependencies = [
+        ("cms", "0028_course_program_index_pages"),
+        ("wagtailcore", "0043_lock_fields"),
```

Comment:
Is there any particular reason for adding this? 

I am getting incosistent history error.
```
web_1     | django.db.migrations.exceptions.InconsistentMigrationHistory: Migration cms.0029_setup_course_program_index_pages is applied before its dependency wagtailcore.0043_lock_fields on database 'default'.
```

Reply:
This was to account for a field name change in one of the Wagtail models, but I forgot to test in cases where you're not migrating from zero. I think I need to change this so it just uses some if/else logic to refer to the model field rather than adding that migration as a dependency.